### PR TITLE
emoji panel revamp

### PR DESCRIPTION
**Pull request by VirajDilshanKumarage**
emoji panel revamp

any issue contact me via email- viraj123e@gmail.com

### CODE REVIEW

File: react/features/chat/components/web/ChatInput.tsx [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/* eslint-disable linebreak-style */
```
Comment (saghul) [inline react/features/chat/components/web/ChatInput.tsx:1]: Don't do this please. Configure your git client or ediitor to use the appropriate line breaks instead.

**Review by saghul — COMMENTED** [review]
Can you please share a screenshot?

File: react/features/chat/components/web/ChatInput.tsx (hunk 1 above)
Comment (VirajDilshanKumarage) [reply react/features/chat/components/web/ChatInput.tsx:1]: got it, Must I recorrect and commit again?

Comment (VirajDilshanKumarage) [reply react/features/chat/components/web/ChatInput.tsx:1]: image

**Comment by VirajDilshanKumarage** [comment]
Okay, I am very sorry

**Comment by saghul** [comment]
It looks like emoji picker doesn't quite fit in the chat tab, does it?

**Comment by VirajDilshanKumarage** [comment]
yeah, but it can be fixed

**Comment by VirajDilshanKumarage** [comment]
It can be fixed

**Comment by VirajDilshanKumarage** [comment]
image

File: react/features/chat/components/web/ChatInput.tsx (hunk 1 above)
Comment (VirajDilshanKumarage) [reply react/features/chat/components/web/ChatInput.tsx:1]: image

**Pull request closed without merging**